<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferences_dialog</class>
 <widget class="QDialog" name="preferences_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="flow">
        <enum>QListView::TopToBottom</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
       <property name="gridSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <property name="currentRow">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Edison Connection</string>
        </property>
        <property name="icon">
         <iconset resource="../scanview_icons.qrc">
          <normaloff>:/icons/icons/connected.png</normaloff>:/icons/icons/connected.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Directories</string>
        </property>
        <property name="icon">
         <iconset resource="../scanview_icons.qrc">
          <normaloff>:/icons/icons/directories.png</normaloff>:/icons/icons/directories.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="conn">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout_3">
           <item row="3" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Edison Username</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="le_username">
             <property name="text">
              <string>root</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Edison Password</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="le_password">
             <property name="text">
              <string>abcd1234</string>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::PasswordEchoOnEdit</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Host Name</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="le_host_name">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>edison</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Ctrlmod Port</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="sb_port">
             <property name="minimum">
              <number>1024</number>
             </property>
             <property name="maximum">
              <number>65536</number>
             </property>
             <property name="value">
              <number>13366</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Edserver Port</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="sb_edserver_port">
             <property name="minimum">
              <number>1024</number>
             </property>
             <property name="maximum">
              <number>65536</number>
             </property>
             <property name="value">
              <number>13367</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="gb_host_ip">
           <property name="title">
            <string>Use IP instead of host name</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Host IP</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="le_host_ip">
              <property name="text">
               <string>192.168.42.1</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="btn_load_config">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../scanview_icons.qrc">
               <normaloff>:/toolbars/icons/toolbars/load_config.png</normaloff>:/toolbars/icons/toolbars/load_config.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btn_save_config">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../scanview_icons.qrc">
               <normaloff>:/toolbars/icons/toolbars/save_cfg.png</normaloff>:/toolbars/icons/toolbars/save_cfg.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="dirs">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Edison Firmware</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLineEdit" name="le_edison_firmware">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbtn_edison_logs_2">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Config Files</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="le_config_files">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="tbtn_config_files">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>Edison Logs</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="le_edison_log"/>
            </item>
            <item>
             <widget class="QToolButton" name="tbtn_edison_logs">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>213</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../scanview_icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gb_host_ip</sender>
   <signal>toggled(bool)</signal>
   <receiver>preferences_dialog</receiver>
   <slot>on_toggle_use_ip(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>408</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbtn_config_files</sender>
   <signal>pressed()</signal>
   <receiver>preferences_dialog</receiver>
   <slot>on_config_files_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>558</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbtn_edison_logs</sender>
   <signal>pressed()</signal>
   <receiver>preferences_dialog</receiver>
   <slot>on_edison_logs_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>559</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbtn_edison_logs_2</sender>
   <signal>pressed()</signal>
   <receiver>preferences_dialog</receiver>
   <slot>on_edison_firmware_btn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>570</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_toggle_use_ip(bool)</slot>
  <slot>on_config_files_btn()</slot>
  <slot>on_edison_logs_btn()</slot>
  <slot>on_edison_firmware_btn()</slot>
 </slots>
</ui>
